qt/python/mantidqt/utils/qt/__init__.py

```diff
@@ -51,7 +51,11 @@ def import_qt(modulename, package, attr=None):
         try:
             lib = import_module(modulename + LIB_SUFFIX, package)
         except ImportError:
-            lib = import_module(modulename.lstrip('.') + LIB_SUFFIX)
+            try:
+                lib = import_module(modulename.lstrip('.') + LIB_SUFFIX)
+            except ImportError:
```

Comment:
I think this could be caused something other than just the module missing. Can we just say a more general import problem and append the two exceptions that we have trapped so far?